### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by lakshyakumar**
**Changelog**
- Fix enode external url
- Update the checks and implement the shared checks.

 

**Reviewed by**
 @suvajit-sarkar 

 

**Linked issue**
#issue_number #590 and #656 

 

 

Commit message
-------------
Implement shared checks and fix enode external url

### CODE REVIEW

File: examples/supplychain-app/configuration/roles/create/quorum/expressapi/tasks/main.yaml [hunk 1]
```diff
@@ -1,4 +1,18 @@
 ############################################################################################
+#Check if the pods for the organizations are up and running
+- name: "Waiting for peer pod {{ peer_data.name }} in {{ organization_data.name | lower }}-ns"
```
Comment (sownak) [inline examples/supplychain-app/configuration/roles/create/quorum/expressapi/tasks/main.yaml:3]: use -quo for namespaces in quorum.

File: examples/supplychain-app/configuration/roles/deploy/quorum/smartContract/tasks/nested_deploy.yaml [hunk 2]
```diff
@@ -1,4 +1,15 @@
 ############################################################################################
+# Check if the peer is up
+- name: "Waiting for peer pod {{ peer.name }} in {{ organization_data.name | lower }}-ns"
```
Comment (sownak) [inline examples/supplychain-app/configuration/roles/deploy/quorum/smartContract/tasks/nested_deploy.yaml:3]: same use -quo

File: platforms/quorum/configuration/roles/create/constellation/tasks/main.yaml [hunk 3]
```diff
@@ -25,7 +25,7 @@
 # This task adds a new node to the existing network
 - name: Adds a new node to the existing network
   shell: |
-    curl -X POST http://{{ geth_data.url }}:{{ geth_data.rpcport }} --data '{"jsonrpc":"2.0","method":"raft_addPeer","params": ["enode://{{ enode.enodeval }}@{{ enode.peer_name }}.{{ external_url }}:{{ enode.p2p_ambassador }}?discport=0&raftport={{ enode.raft_ambassador }}"], "id":{{ geth_data.nodeid }} }' --header "Content-Type: application/json"
+    curl -X POST http://{{ geth_data.url }}:{{ geth_data.rpcport }} --data '{"jsonrpc":"2.0","method":"raft_addPeer","params": ["enode://{{ enode.enodeval }}@{{ enode.peer_name }}.{{ enode.external_url }}:{{ enode.p2p_ambassador }}?discport=0&raftport={{ enode.raft_ambassador }}"], "id":{{ geth_data.nodeid }} }' --header "Content-Type: application/json"
```
Comment (suvajit-sarkar) [inline platforms/quorum/configuration/roles/create/constellation/tasks/main.yaml:28]: use uri ansible module  instead of shell

Comment (sownak) [reply platforms/quorum/configuration/roles/create/constellation/tasks/main.yaml:28]: Can this be addressed? @lakshyakumar

**Pull request merged**